docs/content/concepts/assets/software-defined-assets.mdx

```diff
@@ -377,6 +377,73 @@ def repo():
 
 Like regular jobs, asset jobs can be placed on [schedules](/concepts/partitions-schedules-sensors/schedules) and [sensors](/concepts/partitions-schedules-sensors/sensors).
 
+---
+
+## Grouping assets
+
+To help keep your assets tidy, you can organize them into groups. Grouping assets by project, concept, and so on simplifies [keeping track of them in Dagit](#viewing-asset-groups-in-dagit).
+
+- [Assigning assets to groups](#assigning-assets-to-groups)
+- [Viewing asset groups in Dagit](#viewing-asset-groups-in-dagit)
+
+### Assigning assets to groups
+
+In Dagster, there are two ways to assign assets to groups:
+
+- [By using the `load_assets_from_package_module` function](#from-assets-in-a-sub-module) (**recommended**)
+- [By specifying a group name on an individual asset](#on-individual-assets)
+
+By default, assets that aren't assigned to a group will be placed in a group named `default`. Use Dagit to [view these assets](#viewing-asset-groups-in-dagit).
+
+#### From assets in a sub-module
+
+**This recommended approach** constructs a group of assets from a specified module in your project. Using the `load_assets_from_package_module` function, you can import all assets in a module and apply a grouping:
```

Comment:
Need the actual name of the file here - I wrote down `repo.py`, but wasn't sure if that was right.

Reply:
where will the name of the file be used?

If it's just a hint as to where this code should live, I think `repo.py` will do the job, as that's a common name people will use for the file that holds their repository definition (and really this can be called anywhere if people want, so it's ok to not be super explicit)